modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C

```diff
@@ -12,6 +12,7 @@ InputParameters
 validParams<PorousFlowRelativePermeabilityBase>()
 {
   InputParameters params = validParams<PorousFlowMaterialBase>();
+  params.addParam<Real>("scaling", 1.0, "Relative permeability is multiplied by this factor");
```

Comment:
Nah, i kind of want scaling >= 0, for generality.  Eg, sometimes i suspect that the insitu permeability has been measured as the permeability for water, and then gas permeability might have scaling > 1